packages/compiler-cli/src/ngtsc/typecheck/src/checker.ts

```diff
@@ -712,18 +734,22 @@ export class TemplateTypeCheckerImpl implements TemplateTypeChecker {
   }
 
   getPotentialImportsFor(
-      toImport: PotentialDirective|PotentialPipe,
-      inContext: ts.ClassDeclaration): ReadonlyArray<PotentialImport> {
+      toImport: Reference<ClassDeclaration>, inContext: ts.ClassDeclaration,
+      forceStandaloneImport = false): ReadonlyArray<PotentialImport> {
```

Comment:
```suggestion
      forceStandaloneImport): ReadonlyArray<PotentialImport> {
```
General style for the compiler is to avoid default values for parameters.

Instead of a boolean parameter, you might consider something like:

```typescript
export enum PotentialImportMode {
  Normal,
  ForceDirect,
}
```

as it carries more semantic information than a boolean flag.